### PR TITLE
VZ-6074 fixes to vz upgrade

### PR DESCRIPTION
**Pull request by mgianatagh**
# Description

Fixes to vz upgrade

Fixes VZ-6074

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tools/vz/cmd/helpers/vpo.go [hunk 1]
```diff
@@ -188,39 +194,53 @@ func WaitForOperationToComplete(client clipkg.Client, kubeClient kubernetes.Inte
...
 			for _, condition := range vz.Status.Conditions {
 				// Operation condition met for install/upgrade
 				if condition.Type == condType {
 					resChan <- nil
+					return
 				}
 			}
+
+			// Pause before next status check
+			time.Sleep(15 * time.Second)
```
Comment (bobdonat) [inline tools/vz/cmd/helpers/vpo.go:243]: Seems like a long time to pause for next status check.  Install/upgrade could take up to 15 seconds longer.

File: tools/vz/cmd/upgrade/upgrade.go [hunk 2]
```diff
@@ -93,24 +88,22 @@ func runCmdUpgrade(cmd *cobra.Command, vzHelper helpers.VZHelper) error {
 		return err
 	}
 
-	// When --operator-file is not used, get the version from the command line
-	var version string
```
Comment (bobdonat) [inline tools/vz/cmd/upgrade/upgrade.go:97]: this was the same code for install.  why is upgrade different

File: tools/vz/cmd/helpers/vpo.go (hunk 1 above)
Comment (mgianatagh) [reply tools/vz/cmd/helpers/vpo.go:243]: Should I reduce it to check every second?

Comment (bobdonat) [reply tools/vz/cmd/helpers/vpo.go:243]: That sounds better to me

File: tools/vz/cmd/upgrade/upgrade.go (hunk 2 above)
Comment (mgianatagh) [reply tools/vz/cmd/upgrade/upgrade.go:97]: For install we don't need to know the version up front. For upgrade we need to know what version to put in the Verrazano CR. If just the operator.yaml is supplied, I am not sure if we can determine the version.

Comment (bobdonat) [reply tools/vz/cmd/upgrade/upgrade.go:97]: Maybe we should discuss this more but not hold up the PR

**Review by bobdonat — APPROVED** [review]

**Review by stevshap — APPROVED** [review]
lgtm

**Pull request merged**